src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/References/SetAttributeCommand.cs

```diff
@@ -9,8 +9,8 @@ internal class SetAttributeCommand : SetTreatAsUsedAttributeCommand
         public SetAttributeCommand(AbstractReferenceHandler abstractReferenceHandler, ConfiguredProject selectedConfiguredProject, string itemSpecification)
         : base(abstractReferenceHandler, selectedConfiguredProject, itemSpecification)
         {
-            UnsetValue = PropertySerializer.SimpleTypes.ToString(true);
-            SetValue = PropertySerializer.SimpleTypes.ToString(false);
+            UnsetValue = PropertySerializer.SimpleTypes.ToString(false);
+            SetValue = PropertySerializer.SimpleTypes.ToString(true);
```

Comment:
It would be good to add the parameter name to the value so it is more obvious what the boolean provided does.

Reply:
done